proxy.go

```diff
@@ -6,6 +6,7 @@ import (
 	"net/http"
 	"net/http/httputil"
 	"time"
+    "strings"
```

Comment:
@garbas best to resolve by running `go fmt`, which will also fix other spacing issues lower down